base/server/python/pki/server/instance.py

```diff
@@ -876,10 +876,20 @@ def create(cls, name):
             instance_type = parts[0]
             instance_name = parts[1]
 
+        sysconfig_file = os.path.join('/etc/sysconfig', instance_name)
+
+        with open(sysconfig_file) as f:
+            # Only the value is needed
+            nuxwdog_status = re.search("^USE_NUXWDOG=(.*)", f.read()).group(1)
+
+            # The returned value contains the string: "true" | "false"
+            if "true" in nuxwdog_status:
```

Comment:
The reason I used `in` was because the return literal matched `"true"` (including double quotes). In my latest amend I updated the regex group to only include the value.